Connecting to Server...

Connected!

Listening for messages!

**[RECEIVED]**: 427 bytes
**[RECEIVED]**: {
  "code": 0,
  "modulus": "qDYdf3af9xTVoPLgldI8NtqO1R5jmS8rJ5LLccVmUv99lF1hOiwxRVVAcAlTVWarOjxSI5FXWFXkyAKYAXqzWEG1F10Z40Jisr3TKmm5rNDGOWObR\u002BneONVGUV3LAz671iyZjLFRRdhyudgg4nfOHl0\u002BUY7a2NKllup6CUKkzl\u002BBV0egPl6aq/3xq3GfM8s/JWflZICw2nzsuhVa6GRcaAK2ZSDzg/kxEwWbSXam7IytPmp4\u002BIzw/4SP\u002BVQKSYwwAivy9e/KDVYWAT6nwtfyb6DYKJmD/f77YdrDaaBxpEiCTo0rrLg8DohjS0KgrF3X1SnpUlBdQD8tze9Fx/yxiQ==",
  "exponent": "AQAB"
}

Received RSA. Sending Aes now!

**[SENDING(RSA)]**: 256 bytes
**[SENDING(RSA)]**: {
  "code": 1,
  "key": "ayi6lH1agv9tptshtsFnuPShenea9Qn4PUlB5jAY100=",
  "iv": "uKfOaIoBo3Hf5hGlKmqu3A=="
}

**[RECEIVED]**: 32 bytes
**[RECEIVED]**: {
  "code": 2
}

Server is ready to receive data

**[SENDING(Aes)]**: 128 bytes
**[SENDING(Aes)]**: {
  "code": 9,
  "token": "ZHNmc0BkYXNkZi5jb20=dq7DP00l7iRMcuCJnS9aCP/mF2vRaCBxf7h1PO80u6tpaQx02Nil2rpbjF/0/i4X"
}

**[RECEIVED]**: 848 bytes
**[RECEIVED]**: {
  "code": 10,
  "npgsqlException": {
    "Exception": 0,
    "ColumnName": ""
  },
  "user": {
    "Username": "dafsa",
    "HashTag": "#dsaf",
    "Email": "dsfs@dasdf.com",
    "Password": "",
    "Biography": "SDAFDSA",
    "Id": "2",
    "Birthday": "01.01.2020",
    "ProfilePicture": "**[Image]**",
    "FaEnabled": "True"
  }
}

Received answer to login from server

**[SENDING(Aes)]**: 896 bytes
**[SENDING(Aes)]**: {
  "code": 11,
  "relationshipUpdate": {
    "User": {
      "Username": "dafsa",
      "HashTag": "#dsaf",
      "Email": "dsfs@dasdf.com",
      "Password": "",
      "Biography": "SDAFDSA",
      "Id": "2",
      "Birthday": "01.01.2020",
      "ProfilePicture": "**[Image]**",
      "FaEnabled": "True"
    },
    "Relationship": {
      "Username": "yxc",
      "HashTag": "#asd",
      "Biography": "",
      "Id": "-1"
    },
    "RequestedRelationshipstate": 0
  }
}

ERROR in file Logger.cs, in Log(), at line: 142, at column: 13
ERROR: The process cannot access the file 'C:\Users\Praktikum\source\repos\Cristiano3120\ClientMessenger\ClientMessenger\Logging\12-02-2025.15-56-33.md' because it is being used by another process.

